src/ConstexprMap.h

```diff
@@ -201,7 +196,7 @@ class cConstexprMap
 			}
 		);
 
-		#if defined(DEBUG) && defined(CAN_THROW_IN_CONSTEXPR)
+		#if defined(CAN_THROW_IN_CONSTEXPR)
```

Comment:
Our build system isn't defining `DEBUG`, so my test wasn't being run. `_DEBUG` wasn't defined either and that one is definitely used throughout Cuberite. This seems fishy.

https://github.com/cuberite/cuberite/search?q=_DEBUG